runtime/compiler/env/J9SharedCache.cpp

```diff
@@ -96,6 +103,15 @@ TR_YesNoMaybe TR_J9SharedCache::isSharedCacheDisabledBecauseFull(TR::Compilation
    return _sharedCacheDisabledBecauseFull;
    }
 
+bool
+TR_J9SharedCache::initClassChainValidationBitVectors()
```

Comment:
I haven't checked, but if these vectors are only initialized during `aboutToBootstrap`, then the monitor is not needed because `aboutToBootstrap` is only executed by a single thread before any Java code can run.

Reply:
yeah they're only initialized in `aboutToBootstrap`; I'll change it so we don't acquire the monitor.